ci-operator/config/openstack-k8s-operators/octavia-operator/OWNERS

```diff
@@ -0,0 +1,16 @@
+approvers:
+- abays
+- dprince
+- olliewalsh
```

Comment:
Please add weinimo and beagles in OWNERS file.

Reply:
Added. Thanks.